docs/features/accessibility.md

```diff
@@ -163,6 +163,47 @@ These keyboard shortcuts allow for quick access to content editing features.
 	</tbody>
 </table>
 
+#### Keystrokes for interacting with annotation threads (e.g. comments, track changes suggestions)
+
+<table>
+	<thead>
+		<tr>
+			<th>Action</th>
+			<th>PC</th>
+			<th>Mac</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>Move focus to the thread when the selection is anchored in its marker</td>
+			<td><kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>E</kbd></td>
+			<td><kbd>⌘⇧E</kbd></td>
+		</tr>
+		<tr>
+			<td>Exit the annotation and move focus back to the edited content</td>
+			<td colspan="2"><kbd>Esc</kbd></td>
+		</tr>
+		<tr>
+			<td>Browse the focused annotation thread or thread comment</td>
+			<td colspan="2"><kbd>Enter</kbd></td>
+		</tr>
+		<tr>
+			<td>Move across internals of the annotation thread</td>
```

Comment:
I miss also an info that you can go "up" with Esc. Probably after this entry.

Reply:
See https://github.com/ckeditor/ckeditor5/pull/16861#discussion_r1703928823

Reply:
Also: I think the Esc key navigation is universal across the entire web system so it's not worth a dedicated entry here. Enter, on the other hand, is not the most obvious pattern that we used, that's why it's here. I'm not extremely attached to this strategy, though.